pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+### Description
```

Comment:
I removed this file from dev branch because the right place for it is main branch. Could you please be sure that this file will not be added to dev branch again?

Reply:
✅Done